frontend/nuxt.config.ts

```diff
@@ -27,10 +28,16 @@ export default defineNuxtConfig({
     },
     css: {
       devSourcemap: true,
+      preprocessorOptions: {
+        scss: {
+          api: 'modern-compiler',
+          loadPaths: ['.', 'node_modules'],
+        }
```

Comment:
Addding this to vite.css.preprocessorOptions.scss removes the hundreds of warnings related to UU-Bootstrap.

```suggestion
    silenceDeprecations: [
    "mixed-decls",
    "function-units",
    "global-builtin",
    "import",
    "color-functions",
    ],
```

Reply:
We should remove these after UU-Bootstrap is fixed (and perhaps create an issue for this so we won't forget).

Reply:
`silenceDeprecation` does not work for me, but I have added `quietDeps: true` to silence warnings from dependencies and I have changed our own imports to `use`, which works fine.